docs/Automation.md

```diff
@@ -1,183 +1,196 @@
+        // Optional: configure to create an A11yTest file
+        myConfigBuilder.WithOutputFileFormat(OutputFileFormat.A11yTest);
+
+        // Optional: configure to output the file to a specific directory (otherwise, current directory will be used)
+        myConfigBuilder.WithOutputFileFormat(OutputFileFormat.A11yTest);
+
+        // Ready to use config
+        var myConfig = myConfigBuilder.build();
+
+2. Create a `Scanner` object using the `ScannerFactory` object with the `Config`.
+
+        // Create scanner using myConfig
+        var scanner = ScannerFactory.CreateScanner(myConfig);
+
+3. Call  the `Scan` method on the `Scanner` object.
+
+        var scanResults;
+        try {
+            scanResults = scanner.Scan();
+        }
+        catch(AxeWindowsAutomationException e) {
+            // Get the message from an exception, if one is thrown.
+            var errorMessage = e.InnerException != null ? e.InnerException.Message : e.Message;
+        }
+
+4. Check the results.
 
-###### PowerShell
+        Console.WriteLine("Number of errors found in scan: " + scanResults.errorCount);
 
-The **Start-AxeWindows** cmdlet accepts the following optional
-parameters:
+
+A [complete code example](#example) can be found below.
+
+### Class Details
+
+#### Config.Builder
+
+##### ForProcessId (static)
+Create the builder for the config for the specified process.
+
+##### Parameters
+
+The **ForProcessId** method accepts the following parameters:
 
 **Name** | **Type** | **Description**
 ---|---|---
-OutputFile | `string`  | The location where output from this session will be written. The file extension can (and should) be omitted. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-OutputFileFormat | `string` | The output file format to generate. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-TargetProcessId | `string` | The process id, expressed as a string, that will be scanned. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-ConfigFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Must point to a valid JSON-serialized file if specified.
+processId | `int` | The process Id of the application to test. If the value is invalid, the automation session will throw an [`AxeWindowsAutomationException`](#error-handling).
 
 ##### Return object
 
-The **StartCommandObject** has the following properties:
+The **ForProcessId** method returns an instance of **Config.Builder**.
 
-**Name** | **Type** | **Description**
----|---|---
-Completed | `bool` | True if the command ran to completion. If true, then the Succeeded property indicates if the call succeeded.
-SummaryMessage | `string` | A human-readable message to summarize the result. This message can change at any time, so do not rely on parsing it in tools.
-Succeeded | `bool` | True if the Start command completed successfully.
+##### WithOutputFileFormat
```

Comment:
If it's not possible that's alright, but it would be nice for the methods to be larger/bolder than their attributes (parameter, return, etc)